Packs/Code42/ReleaseNotes/6_0_0.md

```diff
@@ -0,0 +1,49 @@
+
+#### Integrations
+
+##### Code42 Event Collector
+
+- Updated the Docker image to: *demisto/py42:1.0.0.117258*.
+
+##### Code42
+- Updated the Docker image to: *demisto/py42:1.0.0.117258*.
+
+- Added API Gateway URL configuration field for the Code42 integration. This is necessary to support alert sessions.
+- Updated incident fetch to use alert sessions instead of legacy alerts.
+- Updated `code42-alert-get` to use alert sessions instead of legacy alerts.
+- Changed `code42-alert-resolve` to `code42-alert-update`, which can set the alert state to OPEN, CLOSED_TP, or CLOSED_FP
+- Removed deprecated commands:
+  - `code42-securitydata-search` - use `code42-file-events-search` instead.
+  - `code42-departingemployee-add` - use `code42-watchlists-add-user` instead.
+  - `code42-departingemployee-remove` - use `code42-watchlists-remove-user` instead.
+  - `code42-departingemployee-get-all` - use `code42-watchlists-list-included-users` instead.
+  - `code42-departingemployee-get` - use `code42-watchlists-list-included-users` instead.
+  - `code42-highriskemployee-add` - use `code42-watchlists-add-user` instead.
+  - `code42-highriskemployee-remove` - use `code42-watchlists-remove-user` instead.
+  - `code42-highriskemployee-get-all` - use `code42-watchlists-list-included-users` instead.
+  - `code42-highriskemployee-get` - use `code42-watchlists-list-included-users` instead.
+  - `code42-highriskemployee-add-risk-tags` - risk tags are replaced by watchlists.
+  - `code42-highriskemployee-remove-risk-tags` - risk tags are replaced by watchlists.
+- Updated `code42-user-get-risk-profile` and `code42-user-update-risk-profile` to use Actor APIs instead of deprecated risk profile APIs.
+
+
+#### Mappers
+
+##### Code42 - Mapping
+
+- Updated the Code42 mapper to support the new alerts format.
+
+#### Playbooks
+
+##### Code42 Exfiltration Playbook
+
+- Remove V1 file events branch, as it can no longer be triggered.
+- Add the ability to close a Code42 alert as true positive or false positive.
+
+#### Scripts
+
+##### Code42FileEventsToMarkdownTable
+
+- %%UPDATE_RN%%
```

Comment:
Please fill in or remove the %%UPDATE_RN%% line before submitting. This placeholder is used for adding release notes, and leaving it in the final version may cause confusion.